packages/v4/patternfly-docs/content/design-guidelines/components/time picker/time-picker.md

```diff
@@ -38,13 +38,13 @@ Until the user selects a start time, the end time field will be disabled.
 ### Date and time range selection
 Combine a date picker and time picker when you want users to select a range of times across multiple days. 
 
-For more information about using date and time pickers together, visit our [date picker guidelines](/components/date-picker/design-guidelines).
+For more information about using date and time pickers together, visit our [date picker guidelines](/components/date-and-time/date-picker).
```

Comment:
I think this url should be `/components/date-and-time/date-picker/design-guidelines`